apps/trading/pages/markets/trade-grid.tsx

```diff
@@ -53,6 +55,56 @@ const TradingViews = {
 
 type TradingView = keyof typeof TradingViews;
 
+type GetExpiryProps = Pick<TradeMarketHeaderProps, 'market'> & {
```

Comment:
suggestion (non blocking): not sure about the naming and types here. I'd normally reserve naming something XProps for components only. Also I find using `Pick` a bit overkill and hard to read. Additionally would it not be helpful to make getExpiryProps a hook so its visible in dev tools?

I think this would be easier to read
```
const useExpiryProps = ({ market, explorerUrl }: {
  market: TradeMarketHeaderProps['market']
  explorerUrl?: string
}) => { // Note let TS infer return type
  ...
}
```

Reply:
haven't thought about this fn as a hook, but yeah, totally right, where it's called makes it one. since i'm not a huge fan of returning react components / elements from random functions, did a bit of refactor so make these items individual react components. let me know if this works.